src/Parallelization.php

```diff
@@ -389,6 +389,7 @@ protected function executeMasterProcess(InputInterface $input, OutputInterface $
                     '--child',
                     '--env='.$input->getOption('env'),
                     $input->getOption('no-debug') ? '--no-debug' : '',
+                    implode(" ", $this->serializeInputOptions($input, ['child','env','no-debug', 'processes'])),
```

Comment:
We don't need to implode here since it's done just above, so instead we can just append those elements:

```suggestion
                    ...$this->serializeInputOptions($input, ['child','env','no-debug', 'processes']),
```

Reply:
I am not sure if there is an easier way or if I missed your point, but i used array_merge to add the options.